src/app/providers/config/ServiceConstantsContextProvider.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { ConstantContext, ServiceConstants } from './ServiceConstants';
+import constantsFallback   from '../../../../static/configs/service-constants.json'
+declare const __webpack_public_path__: string;
+
+export const ServiceConstantsContextProvider: React.FunctionComponent = ({ children }) => {
+  const [value, setValue] = React.useState<ServiceConstants | undefined>(undefined);
+
+  React.useEffect(() => {
+    (async () => {
+      if (value === undefined) {
+        const response = await fetch(`${__webpack_public_path__}service-constants.json`);
+        try {
+          const jsonConfig = await response.json();
+          console.debug('Done loading constants');
+          setValue(jsonConfig);
+        } catch (error) {
+          console.debug('Using embedded constants as fallback');
+          // Use embeeded config as fallback
+          setValue(constantsFallback);
```

Comment:
What's the purpose of this? A fallback a built in file to a remote URL makes sense (try the fast approach, fallback to the slower), but falling back from a slow approach to a fast approach is odd.